src/pcapi/core/users/repository.py

```diff
@@ -13,6 +13,8 @@ def get_user_with_credentials(identifier: str, password: str) -> UserSQLEntity:
     user = find_user_by_email(identifier)
     if not user:
         raise exceptions.InvalidIdentifier()
+    if not user.isActive:
+        raise exceptions.InvalidIdentifier()
```

Comment:
```suggestion
    if not user or not user.isActive:
        raise exceptions.InvalidIdentifier()
```
C'est une proposition :)

Reply:
Je voulais que ce soit plus explicite, mais oui, pourquoi pas.

Reply:
Ok, pas de problèmes :)